Formula/llvm.rb

```diff
@@ -133,8 +164,8 @@ def install
       # Workaround for CMake Error: failed to create symbolic link
       ENV.deparallelize if Hardware::CPU.arm?
       system "cmake", "--build", "."
-      system "cmake", "--build", ".", "--target", "install"
-      system "cmake", "--build", ".", "--target", "install-xcode-toolchain" if MacOS::Xcode.installed?
+      system "cmake", "--build", ".", "--target", "install/strip"
+      system "cmake", "--build", ".", "--target", "install-xcode-toolchain/strip" if MacOS::Xcode.installed?
```

Comment:
Oops:
```
make: *** No rule to make target `install-xcode-toolchain/strip'.  Stop.
```
They have `/strip` targets for all their other `install` targets; I would've thought they'd have one for this too.

Reply:
Yeah that's a bummer.  At least we have the answer now.  Do you want me to wait to push to this again until you've had a chance to test your LLVM PR?

Reply:
I'm probably going to sit on that for a while, but if could pull in the cosmetic changes from there to here (i.e. the ones that don't involve `C_INCLUDE_DIRS`) I'd appreciate it.

Reply:
i.e. These:
```diff
diff --git a/Formula/llvm.rb b/Formula/llvm.rb
index 1bbe232ae6..a0bc619c42 100644
--- a/Formula/llvm.rb
+++ b/Formula/llvm.rb
@@ -76,7 +76,7 @@ class Llvm < Formula
 
     # we install the lldb Python module into libexec to prevent users from
     # accidentally importing it with a non-Homebrew Python or a Homebrew Python
-    # in a non-default prefix
+    # in a non-default prefix. See https://lldb.llvm.org/resources/caveats.html
     args = %W[
       -DLLVM_ENABLE_PROJECTS=#{projects.join(";")}
       -DLLVM_ENABLE_RUNTIMES=#{runtimes.join(";")}
@@ -99,6 +99,9 @@ class Llvm < Formula
       -DLLDB_PYTHON_RELATIVE_PATH=libexec/#{site_packages}
       -DLIBOMP_INSTALL_ALIASES=OFF
       -DCLANG_PYTHON_BINDINGS_VERSIONS=#{py_ver}
+      -DPACKAGE_VENDOR=#{tap.user}
+      -DPACKAGE_BUGREPORT=#{tap.issues_url}
+      -DCLANG_VENDOR_UTI=org.#{tap.user.downcase}.clang
     ]
 
     if MacOS.version >= :catalina
@@ -113,7 +116,7 @@ class Llvm < Formula
       args << "-DLLVM_BUILD_LLVM_C_DYLIB=ON"
       args << "-DLLVM_ENABLE_LIBCXX=ON"
       args << "-DLLVM_CREATE_XCODE_TOOLCHAIN=#{MacOS::Xcode.installed? ? "ON" : "OFF"}"
-      args << "-DRUNTIMES_CMAKE_ARGS=-DCMAKE_INSTALL_RPATH=@loader_path/../lib"
+      args << "-DRUNTIMES_CMAKE_ARGS=-DCMAKE_INSTALL_RPATH=#{rpath}"
 
       sdk = MacOS.sdk_path_if_needed
       args << "-DDEFAULT_SYSROOT=#{sdk}" if sdk
```

Reply:
Actually I see you marked that PR as a draft now.  I'll go ahead and push the fix for this, but feel free to cancel my run if it's causing problems.

Reply:
Yup, I'll need to test that one locally a bit first. But if you could include some of the other changes (see above) here to save some CI runs, that'd be great.

Reply:
I think I got all the changes in now.  If you see anything wrong or missing I'll restart the job rather than wait.